kb/plane_geometry/formal_statements/statements/statements_scs/stat_rectangular_isosceles_triangle_median_cathetus.scs

```diff
@@ -0,0 +1,14 @@
+stat_rectangular_isosceles_triangle_median_cathetus <- statement;;
+
+stat_rectangular_isosceles_triangle_median_cathetus -> 	rrel_key_sc_element: concept_rectangular_triangle; concept_isosceles_triangle; nrel_median; nrel_cathetus;;
+
+stat_rectangular_isosceles_triangle_median_cathetus => nrel_main_idtf: [Акс.(прямоугольный треугольник;равнобедренный треугольник;катет*;медиана*)] (*<- lang_ru;;*);;
+stat_rectangular_isosceles_triangle_median_cathetus => nrel_main_idtf: [Ax.(rectangular triangle;isosceles triangle;median*;cathetus*)] (*<- lang_en;;*);;

```

Comment:
It isn't an axiom. It is a statement. You should use [Stat.(...)] insted of [Ax.(...)]